cmd/hlb/command/run.go

```diff
@@ -222,14 +233,14 @@ func Run(ctx context.Context, cln *client.Client, rc io.ReadCloser, info RunInfo
 		return err
 	}
 
-	if solveReq == nil || info.Debug || info.Tree {
```

Comment:
I noticed that this caused hlb to exit early when debugging was turned on. This seems counter-intuitive to me, because I'm not sure why using the debugger means we want to skip the solve. I removed this for now, but please let me know if I'm missing something.

Reply:
In the beginning, it was useful to just debug the debugger without having to cancel the solve every time, so this is just leftover from that. Hopefully HLB will one day execute in a single gateway build function, in which case solving every time is the only option.